specification/indices/_types/IndexSettings.ts

```diff
@@ -269,3 +383,119 @@ export class Queries {
 export class CacheQueries {
   enabled: boolean
 }
+
+/**
+ * Mapping Limit Settings
+ * @doc_url https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-settings-limit.html
```

Comment:
Can you create a `doc_id` entry for this?
https://github.com/elastic/elasticsearch-specification/blob/main/specification/_doc_ids/table.csv